src/styles/service_order.css

```
@@ -79,6 +79,10 @@
     padding: 24px;
 }
 
+.service-list {
```

```suggestion
.service-instance-list {
```
I suggest renaming the css name.

---

fixed.